chicken-sloths/bangazon-cli-sprint2
# The Database creation

# Description
I made functions to create all of the Tables that we'll need in the database.
I had to regenerate the data for this PR. They have new properties/names now. So you have to pull down the json files in order for the db to be created.
  
## Related Ticket(s)
Fixes #24

## Problem to Solve
We don't have a database

## Expected Behavior
There is a database with all the correct information

## Steps to Test Solution
1. Regenerate the data first. run `npm run data:generate` FIRST
1. Open the project, look the the `db` folder, there should be our database file in there. 
1b. Make sure there is no sqlite database in the root of our project. check before and after creating the db. 
2. In the terminal, from the project root, run `npm run db:reset`
3. Open that database in sqlectron or dbBrowser, and look through each of the Tables to make sure all the data we need is in the right columns and exists. Double check that the columns we need are actually there.

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods
